src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -230,57 +217,21 @@ Team SensorFusion::createFriendlyTeam(const std::vector<RobotDetection> &robot_d
 {
     Team new_friendly_team =
         friendly_team_filter.getFilteredData(friendly_team, robot_detections);
-    RobotId friendly_goalie_id = DynamicParameters->getAIControlConfig()
-                                     ->getRefboxConfig()
-                                     ->FriendlyGoalieId()
-                                     ->value();
+    RobotId friendly_goalie_id =
+        DynamicParameters->getSensorFusionConfig()->FriendlyGoalieId()->value();
     new_friendly_team.assignGoalie(friendly_goalie_id);
     return new_friendly_team;
 }
 
 Team SensorFusion::createEnemyTeam(const std::vector<RobotDetection> &robot_detections)
 {
     Team new_enemy_team = enemy_team_filter.getFilteredData(enemy_team, robot_detections);
-    RobotId enemy_goalie_id = DynamicParameters->getAIControlConfig()
-                                  ->getRefboxConfig()
-                                  ->EnemyGoalieId()
-                                  ->value();
+    RobotId enemy_goalie_id =
+        DynamicParameters->getSensorFusionConfig()->EnemyGoalieId()->value();
     new_enemy_team.assignGoalie(enemy_goalie_id);
     return new_enemy_team;
 }
 
-bool SensorFusion::isCameraEnabled(const SSL_DetectionFrame &detection)
```

Comment:
Why is this being deleted?

Reply:
Now that Division B effectively has 1 camera, there's no need to be able to disable multiple. We already have functionality to limit the area of the field that we see by coordinates, which is more flexible than disabling entire cameras, so this was no longer really needed.

Reply:
I thought there were two cameras at Robocup 2019

Reply:
2 camera for Divison A, 1 camera for Division B.